src/sdl2_compat.c

```diff
@@ -10487,10 +10487,16 @@ SDL_BuildAudioCVT(SDL_AudioCVT *cvt,
             &ap,
             sizeof(ap));
 
-        cvt->filters[0] = NULL;
+        /* Saurbraten pokes around in undocumented SDL2 internals to see if format conversion
+         * is needed instead of just reading the documented field. Fake a filter in the first
+         * position if we're going to need audio conversion, so apps that do this will work. */
+        cvt->filters[0] = (SDL_AudioFilter)1;
+        cvt->filter_index = 1;
```

Comment:
Saurbraten doesn't look at `filter_index`, but might as well set it if we going to play games with `filters` too...